### ISSUE TITLE
Error: init vault service: vault init: vault not deployed by factory

### ISSUE THREAD

**Issue by asd976385560** [bittorrent/go-btfs#76]
C:\Users\BTT\AppData\Roaming\BitTorrent\btfs>btfs daemon --chain-id 199
Initializing daemon...
go-btfs version: 2.1.0-56bba50
Repo version: 10
System version: amd64/windows
Golang version: go1.15.15
Repo location: E:\btfs_data
Peer identity: 16Uiu2HAm9XyGbH21A96s6
the address of Bttc format is:  0xDFE687526Acf28160
the address of Tron format is:  TWP5rFB8fUtx4kDw
self vault: 0xcc380dd30148cf3
init settlement err:  init vault service: vault init: vault not deployed by factory

Error: init vault service: vault init: vault not deployed by factory

**Comment by Ne2pp** [comment]
I have the same problem, updated the file 2.0.2 to 2.1.0

**Comment by VitaliyAT** [comment]
Plus 1 = me too

**Comment by simbadMarino** [comment]
Guys, please check this issue (already closed) https://github.com/bittorrent/go-btfs/issues/73.

**Comment by asd976385560** [comment]
![image](https://user-images.githubusercontent.com/41860268/154877684-e6927973-3c72-4a28-8687-c7f889b9ef40.png)
![image](https://user-images.githubusercontent.com/41860268/154877746-bc3a5ba6-6bec-43a5-a4f9-1321e10d1a3b.png)

**Comment by simbadMarino** [comment]
> ![image](https://user-images.githubusercontent.com/41860268/154877684-e6927973-3c72-4a28-8687-c7f889b9ef40.png) ![image](https://user-images.githubusercontent.com/41860268/154877746-bc3a5ba6-6bec-43a5-a4f9-1321e10d1a3b.png)

Which command you used to  initialize your btfs node?

**Comment by asd976385560** [comment]
> 
I have just upgraded from version 2.0.2 to 2.1.0 (simply changed over the executable file), and I execute the below command to start the btfs deamon:
btfs daemon --chain-id 199

**Comment by asd976385560** [comment]
> > ![图像](https://user-images.githubusercontent.com/41860268/154877684-e6927973-3c72-4a28-8687-c7f889b9ef40.png) ![图像](https://user-images.githubusercontent.com/41860268/154877746-bc3a5ba6-6bec-43a5-a4f9-1321e10d1a3b.png)
> 
> 您使用哪个命令来初始化 btfs 节点？
I have just upgraded from version 2.0.2 to 2.1.0 (simply changed over the executable file), and I execute the below command to start the btfs deamon:
btfs daemon --chain-id 199

**Comment by simbadMarino** [comment]
Did you do the following? Coming from v1.6 --> v2.02 --> v2.1?
Either way Sophia uploaded a potential solution for this at: https://docs.btfs.io/docs/tutorials-on-upgrading-btfs-v10-to-btfs-v20-mainnet#solutions-to-common-errors.

Give it a try ;)

**Comment by VitaliyAT** [comment]
> Did you do the following? Coming from v1.6 --> v2.02 --> v2.1? Either way Sophia uploaded a potential solution for this at: https://docs.btfs.io/docs/tutorials-on-upgrading-btfs-v10-to-btfs-v20-mainnet#solutions-to-common-errors.
> 
> Give it a try ;)

No, I came just from 2.0.0 - why it should be start from 1.6 ? It's bug by itself "Install DOS first for using Windows" - last time it was in 20th century with Win 3.11 
It's a big shame if such project should be installed by "v1.6 --> v2.02 --> v2.1" ! 
Are we play in homebrew games or this is serious project? - right now look like first.

**Comment by simbadMarino** [comment]
Kindly re-read my response, if you are having problems then try reading as much documentation as you can and keep trying, right now BTFS is not super user friendly but it works, yeah you might need to get your hands dirty a bit but the results are great . Cheers!



Sent from myMail for iOS


Monday, 21 February 2022 22:21 -0600 from ***@***.***  ***@***.***>:
